### PR TITLE
feat(environments): Highlight active environment in dropdown

### PR DESCRIPTION
**Pull request by lynnagara**

### CODE REVIEW

**Review by MaxBittker — APPROVED** [review]
perhaps this would read nicer with the classnames package?

either way, this is a good improvement. 

are there any other edge cases with the empty string environments  that this wouldn't handle correctly?

File: src/sentry/static/sentry/app/components/projectHeader/index.jsx [hunk 1]
```diff
@@ -115,11 +115,20 @@ class ProjectHeader extends React.Component {
...
+                >
                   {allEnvironmentsLabel}
                 </MenuItem>
                 {environments.map(env => (
-                  <MenuItem key={env.id} onClick={() => setActiveEnvironment(env)}>
+                  <MenuItem
+                    key={env.id}
+                    onClick={() => setActiveEnvironment(env)}
+                    className={
+                      activeEnvironment && activeEnvironment.name === env.name && 'active'
```
Comment (MaxBittker) [inline src/sentry/static/sentry/app/components/projectHeader/index.jsx]: what if activeEnvironment is ""? i forget how the weird cases work

Comment (lynnagara) [reply src/sentry/static/sentry/app/components/projectHeader/index.jsx]: It's not an issue here

**Comment by lynnagara** [comment]
We don't need to handle the other cases here, this component just gets passed the already validated environment, and doesn't modify it in anyway

**Pull request merged**